.github/workflows/ci.yml

```diff
@@ -93,3 +94,62 @@ jobs:
         env:
           CI_NAME: github-actions
           COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+
+  publish:
+    if: >-
+        ((github.event_name == 'workflow_dispatch') || (github.event_name == 'push')) &&
+        (github.ref == 'refs/heads/pakrym/release-pipeline' || startsWith(github.ref, 'refs/tags/v')) &&
+        endsWith(github.actor, '-stripe')
+    needs: [build, test]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Setup Java
+        id: setup-jre
+        uses: actions/setup-java@v1
+        with:
+          java-version: "18" # always use 18 for building
+          architecture: x64
+      - name: Set Test Java Runtime Environment variable
+        run: echo "JAVA_TEST_HOME=${{ steps.setup-jre.outputs.path }}" >> $GITHUB_ENV
```

Comment:
I wish we could share this across workflows :')

Reply:
Same.